src/WCSimConstructRealisticPlacement.cc

```diff
@@ -0,0 +1,1168 @@
+//  -*- mode:c++; tab-width:4;  -*-
+#include "WCSimDetectorConstruction.hh"
+
+#include "G4Material.hh"
+#include "G4Element.hh"
+#include "G4Box.hh"
+#include "G4Tubs.hh"
+#include "G4UnionSolid.hh"
+#include "G4Sphere.hh"
+#include "G4Trd.hh"
+#include "G4IntersectionSolid.hh"
+#include "G4Polyhedra.hh"
+#include "G4LogicalVolume.hh"
+#include "G4ThreeVector.hh"
+#include "G4RotationMatrix.hh"
+#include "G4PVReplica.hh"
+#include "G4PVPlacement.hh"
+#include "G4PVParameterised.hh"
+#include "G4AssemblyVolume.hh"
+#include "G4SubtractionSolid.hh"
+#include "globals.hh"
+#include "G4VisAttributes.hh"
+#include "G4LogicalBorderSurface.hh"
+#include "G4LogicalSkinSurface.hh"
+#include "G4OpBoundaryProcess.hh"
+#include "G4OpticalSurface.hh"
+#include "G4UserLimits.hh"
+#include "G4ReflectionFactory.hh"
+#include "G4GeometryTolerance.hh"
+#include "G4GeometryManager.hh"
+
+#include "WCSimTuningParameters.hh" //jl145
+
+#include "G4SystemOfUnits.hh"
+#include "G4PhysicalConstants.hh"
+
+#include "G4VSolid.hh" // following 4 .hh required for CADMesh.
+#include "G4Trd.hh"
+#include "G4NistManager.hh"
+#include "G4AssemblyVolume.hh"
+
+#ifndef DEBUG_RELEASTIC_GEOMETRY
+#define DEBUG_RELEASTIC_GEOMETRY 0
+#endif
+
+struct RealisticPlacementConfiguration {
+
+    G4ThreeVector EntireDetectOffset; //< Relative offset for
+
+    G4Material* InnerDetectorMaterial;
+    G4Material* OuterDetectorMaterial;
+
+    G4VisAttributes* BlackTyvekVis;
```

Comment:
```suggestion
    // Black tyvek is place on the wall of the ID
    G4VisAttributes* BlackTyvekVis;
```
Relatedly, is the black sheet actually tyvek?